### DISCUSSION TITLE
Line Charts | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#642] [Q&A]
# Line Charts | Seeed Studio Wiki

This repo describes how to draw line graphs on Wio Terminal. You can use this to display raw sensor values to visualize readings on screen just like the serial plotter! The library is very flexible and can alter to your own need.

https://wiki.seeedstudio.com/Wio-Terminal-LCD-Linecharts/

**Comment by Mikkel-danielsen** [comment]
How can I use theminimum value from the stored data? I would like to use it as input ".based_on(0.0) //Starting point of y-axis, must be a float"? I would like to have the x-min to be dynamic, just like x-max is.

**Reply by Mikkel-danielsen** [reply]
I tried to read in to the library, but it is to advanced for me. :(

**Comment by Mikkel-danielsen** [comment]
Is it possible to draw to lines in one graph, with two separat data set?
